test/all_functional.spec.ts

```diff
@@ -55,23 +69,28 @@ const modules = modulesList();
 describe('functional tests', () => {
   for (const locale in faker.locales) {
     describe(locale, () => {
-      faker.locale = locale;
       Object.keys(modules).forEach((module) => {
         describe(module, () => {
-          // if there is nothing to test, create a dummy test so the test runner doesn't complain
-          if (Object.keys(modules[module]).length === 0) {
-            it.todo(`${module} was empty`);
-          }
-
           modules[module].forEach((meth) => {
-            it(meth + '()', () => {
+            const testAssertion = () => {
+              faker.locale = locale;
+              faker.seed(1); // TODO: Use random seed once there are no more failures
```

Comment:
Lets use our project-wide TODO format
```suggestion
              faker.seed(1); // TODO ST-DDT 2022-03-25: Use random seed once there are no more failures
```

applies for two other TODOs below